tests/integration/list/stack_outputs/stack_outputs_integ_base.py

```diff
@@ -0,0 +1,45 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+
+
+class StackOutputsIntegBase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cmd = cls.base_command()
+        cls.stack_outputs_test_data_path = Path(__file__).resolve().parents[1].joinpath("testdata", "list")
+
+    def setUp(self):
+        super().setUp()
+        self.scratch_dir = str(Path(__file__).resolve().parent.joinpath(str(uuid.uuid4()).replace("-", "")[:10]))
+        shutil.rmtree(self.scratch_dir, ignore_errors=True)
+        os.mkdir(self.scratch_dir)
+
+        self.working_dir = tempfile.mkdtemp(dir=self.scratch_dir)
```

Comment:
Any directories we make we should cleanup in the `tearDown()` step.

Reply:
added it in